cypress/integration/finance/groups/search-and-gilter.spec.js

```diff
@@ -0,0 +1,79 @@
+import permissions from '../../../support/dictionary/permissions';
+import testType from '../../../support/dictionary/testTypes';
+import devTeams from '../../../support/dictionary/devTeams';
+import FiscalYears from '../../../support/fragments/finance/fiscalYears/fiscalYears';
+import TopMenu from '../../../support/fragments/topMenu';
+import Ledgers from '../../../support/fragments/finance/ledgers/ledgers';
+import Users from '../../../support/fragments/users/users';
+import FinanceHelp from '../../../support/fragments/finance/financeHelper';
+import Groups from '../../../support/fragments/finance/groups/groups';
+
+describe('ui-finance: Groups', () => {
+  const defaultFiscalYear = { ...FiscalYears.defaultUiFiscalYear };
+  const defaultLedger = { ...Ledgers.defaultUiLedger };
+  const defaultGroup = { ...Groups.defaultUiGroup };
+  let user;
+
+  before(() => {
+    cy.getAdminToken();
+    FiscalYears.createViaApi(defaultFiscalYear)
+      .then(response => {
+        defaultFiscalYear.id = response.id;
+        defaultLedger.fiscalYearOneId = defaultFiscalYear.id;
+
+        Ledgers.createViaApi(defaultLedger)
+          .then(ledgerResponse => {
+            defaultLedger.id = ledgerResponse.id;
+
+            Groups.createViaApi(defaultGroup)
+              .then(groupResponse => {
+                defaultGroup.id = groupResponse.id;
+              });
+          });
+      });
+    cy.createTempUser([
+      permissions.uiFinanceCreateViewEditGroups.gui,
+      permissions.uiFinanceViewEditDeletGroups.gui,
```

Comment:
Are you sure that you need these two permissions?
According to the test-case the user should have permission to view groups. There is a permission with name "Finance: View group". Please clarify with PO, manual tester if your permission is valid in this case

Reply:
Deleted